packages/format-library/src/language/index.js

```diff
@@ -13,6 +13,7 @@ import {
 	Button,
 	Popover,
 	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
```

Comment:
## Testing instructions

In the editor, add a Paragraph block and select some of the text. Select "Language" from the block toolbar.

<img src="https://github.com/user-attachments/assets/7997935b-c7ff-42cc-a8dd-c2ec02135364" alt="Screenshot 2024-08-03 at 4 09 04" width="486">

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/553c8963-1cbf-44cb-87d6-e2e9d6e22600" alt="Language popover, before" width="292">|<img src="https://github.com/user-attachments/assets/a0d0af06-ddf5-4b00-b9fc-2c06d945d71f" alt="Language popover, after" width="295">|